gengokit/handler/handler.go

```diff
@@ -283,30 +285,15 @@ func exprString(e ast.Expr) string {
 	return ""
 }
 
-func applyServerTempl(exec *gengokit.TemplateExecutor) (io.Reader, error) {
+func applyServerTempl(exec *gengokit.Executor) (io.Reader, error) {
 	log.Debug("Rendering handler for the first time")
-	return applyTemplate(serverTempl, "ServerTempl", exec)
+	return exec.ApplyTemplate(serverTempl, "ServerTempl")
 }
 
 func applyServerMethsTempl(exec handlerExecutor) (io.Reader, error) {
-	return applyTemplate(serverMethsTempl, "ServerMethsTempl", exec)
-}
-
-func applyTemplate(templ string, templName string, exec interface{}) (io.Reader, error) {
 	funcMap := template.FuncMap{
 		"ToLower": strings.ToLower,
 		"GoName":  generatego.CamelCase,
```

Comment:
One thing I notice is that the same `FuncMap` is set up in a couple places, and it's the same everywhere, except in one place an additional "Titles" function is set up. Why not just declare this `FuncMap` as a package level variable rather than redefining it each time templates are going to be rendered?